.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+# Contributing
+ 
+When contributing to this repository, please first discuss the change you wish to make via issue,
+or [Hypertrace slack](https://bit.ly/hypertrace-slack) with the owners of this repository before making a change.
```

Comment:
I hope this won't expire. Full link might be better.

Reply:
This will expire but we will update from bit.ly dashboard.